### PR TITLE
Dbt worflow

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

Fix https://github.com/open-metadata/OpenMetadata/issues/9024
Fix https://github.com/open-metadata/OpenMetadata/issues/5559
Fix https://github.com/open-metadata/OpenMetadata/issues/7038
Fix https://github.com/open-metadata/OpenMetadata/issues/8535
Fix https://github.com/open-metadata/OpenMetadata/issues/9258

UI : 
- Remove DBT from MetdaData Ingestion
- Added separate workflow for DBT Ingestion

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :
image
image
image
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui 
@open-metadata/backend 
@open-metadata/ingestion

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress.config.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default defineConfig({
   viewportWidth: 1240,
   viewportHeight: 660,
   watchForFileChanges: false,
-  defaultCommandTimeout: 5000,
+  defaultCommandTimeout: 50000,
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress.config.ts:9]: Revert this

File: openmetadata-ui/src/main/resources/ui/cypress.config.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export default defineConfig({
     setupNodeEvents(on, config) {
       return plugins(on, config);
     },
-    baseUrl: 'http://localhost:8585',
+    baseUrl: 'http://localhost:3000',
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress.config.ts:20]: revert this

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js [hunk 3]
```diff
@@ -201,11 +201,11 @@ export const testServiceCreationAndIngestion = (
...
-    if (isDatabaseService(type)) {
-        cy.contains('Configure DBT Model').should('be.visible');
-        configureDBT && configureDBT();
-        cy.get('[data-testid="submit-btn"]').should('be.visible').click();
-    }
+    // if (isDatabaseService(type)) {
+    //     cy.contains('Configure DBT Model').should('be.visible');
+    //     configureDBT && configureDBT();
+    //     cy.get('[data-testid="submit-btn"]').should('be.visible').click();
+    // }
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/common/common.js:208]: Remove this

File: openmetadata-ui/src/main/resources/ui/cypress/common/common.js [hunk 4]
```diff
@@ -135,7 +135,7 @@ export const testServiceCreationAndIngestion = (
     serviceName,
     type = 'database',
     testIngestionButton = true,
-    configureDBT
+    // configureDBT
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/common/common.js:138]: remove this

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/AddNewService/redshiftWithDBT.spec.js [hunk 5]
```diff
@@ -57,24 +56,24 @@ describe('RedShift Ingestion', () => {
...
+    //     .type(HTTP_CONFIG_SOURCE.DBT_CATALOG_HTTP_PATH);
+    //   cy.get('[data-testid="manifest-url"]')
+    //     .scrollIntoView()
+    //     .should('be.visible')
+    //     .type(HTTP_CONFIG_SOURCE.DBT_MANIFEST_HTTP_PATH);
+    //   cy.get('[data-testid="run-result-file"]')
+    //     .scrollIntoView()
+    //     .should('be.visible')
+    //     .type(HTTP_CONFIG_SOURCE.DBT_RUN_RESTLTS_FILE_PATH);
+    // };
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/AddNewService/redshiftWithDBT.spec.js:76]: Remove this

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/AddNewService/redshiftWithDBT.spec.js [hunk 6]
```diff
@@ -83,7 +82,7 @@ describe('RedShift Ingestion', () => {
       REDSHIFT.serviceName,
       'database',
       true,
-      configureDBT
+    //   configureDBT
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/AddNewService/redshiftWithDBT.spec.js:85]: Remove this

File: openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/AddIngestion.component.tsx [hunk 7]
```diff
@@ -779,14 +783,17 @@ const AddIngestion = ({
 
         {activeIngestionStep === 2 && (
           <DBTConfigFormBuilder
-            cancelText="Back"
+            cancelText="Cancel"
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/AddIngestion.component.tsx:786]: localisation

File: openmetadata-ui/src/main/resources/ui/src/components/common/DBTConfigFormBuilder/DBTConfigFormBuilder.test.tsx [hunk 8]
```diff
@@ -148,16 +149,20 @@ const mockCancel = jest.fn();
 const mockSubmit = jest.fn();
 const mockCatalogChange = jest.fn();
 const mockManifestChange = jest.fn();
+const mockIngestionName = jest.fn();
 
 const mockProps = {
   data: mockData,
   okText: 'Next',
-  cancelText: 'Back',
+  cancelText: 'Cancel',
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/DBTConfigFormBuilder/DBTConfigFormBuilder.test.tsx:157]: localisation

File: openmetadata-ui/src/main/resources/ui/src/components/common/DBTConfigFormBuilder/DBTConfigFormBuilder.tsx [hunk 9]
```diff
@@ -238,6 +242,25 @@ const DBTConfigFormBuilder: FunctionComponent<DBTConfigFormProps> = ({
 
   return (
     <Fragment>
+      <Field>
+        <label className="tw-block tw-form-label tw-mb-1" htmlFor="name">
+          Name
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/DBTConfigFormBuilder/DBTConfigFormBuilder.tsx:247]: localisation

File: openmetadata-ui/src/main/resources/ui/src/components/common/DBTConfigFormBuilder/DBTConfigFormBuilder.tsx [hunk 10]
```diff
@@ -238,6 +242,25 @@ const DBTConfigFormBuilder: FunctionComponent<DBTConfigFormProps> = ({
...
+        </label>
+        <p className="tw-text-grey-muted tw-mt-1 tw-mb-2 tw-text-sm">
+          Name that identifies this configuration instance uniquely.
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/DBTConfigFormBuilder/DBTConfigFormBuilder.tsx:250]: localisation

**Review by chirag-madlani — CHANGES_REQUESTED** [review]

**Comment by Ashish8689** [comment]
@chirag-madlani changes made as per comments!

**Review by chirag-madlani — APPROVED** [review]
Looks good on UI side

**Review by chirag-madlani — APPROVED** [review]

**Review by ulixius9 — APPROVED** [review]

**Pull request merged**